### PR TITLE
Feature/dmc update

### PR DESCRIPTION
**Pull request by hferrone**
## Changes
- Fixes: # .

### CODE REVIEW

**Comment by hferrone** [comment]
@keveleigh This is a test branch for a proposed update, just FYI

**Comment by keveleigh** [comment]
@hferrone Ah I missed that, thanks!

**Comment by hferrone** [comment]
@keveleigh I would like to merge it into the feature/dmc branch though

File: Documentation/GettingStartedWithTheMRTK.md [hunk 1]
```diff
@@ -1,213 +0,0 @@
-# Getting started with MRTK
```
Comment (polar-kev) [inline Documentation/GettingStartedWithTheMRTK.md:1]: @hferrone, I thought we agreed we would keep the old Getting StartedWithMRTK.md and have it say that the page has been replaced by installation.md

**Review by keveleigh — APPROVED** [review]

**Pull request merged**